zoslib.gyp

```diff
@@ -32,7 +32,7 @@
           'include_dirs': ['include'],
         }, {
           'cflags': ['-fzos-le-char-mode=ascii', '-fgnu-keywords', '-fno-short-enums', '-mzos-target=zosv2r4'],
-          'include_dirs': ['include', 'include/c++/v1'],
+          'include_dirs': ['include'],
```

Comment:
Can remove the include_dirs on line 32 and move this one to before sources below.

Reply:
CMakeLists.txt also has `include_directories(BEFORE include include/c++/v1)` that should be removed.

Reply:
Thanks.  made the changes.